moveit_core/robot_model/include/moveit/robot_model/joint_model.h

```diff
@@ -469,6 +479,9 @@ class JointModel
 protected:
   void computeVariableBoundsMsg();
 
+  /** \brief The factor applied to the distance between two joint states */
```

Comment:
Because I moved distance_factor toward the front in the initializer list, I also moved the member declaration to comply with [this rule](https://github.com/isocpp/CppCoreGuidelines/blob/036324/CppCoreGuidelines.md#c47-define-and-initialize-member-variables-in-the-order-of-member-declaration).